pikaur/core.py

```diff
@@ -110,9 +110,10 @@ def __repr__(self) -> str:
 
 
 def sudo(cmd: List[str]) -> List[str]:
+    privilege_escalation_tool = PikaurConfig().misc.PrivilegeEscalationTool.get_str()
```

Comment:
makes more sense to define this var after return or even just inline it, as it have only one usage

Reply:
this one still remains to be done